src/engine/init.c

```diff
@@ -1104,6 +1105,17 @@ main(int argc, char **argv)
 	int		sig;
 	int		rc;
 
+	/**
+	 * Disable transparent huge pages (THP) early before any allocations
+ 	 * see https://github.com/pmodels/argobots/issues/369 for more
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs